spyder/plugins/ipythonconsole/tests/test_ipythonconsole.py

```diff
@@ -71,6 +72,22 @@ def get_console_background_color(style_sheet):
     return background_color
 
 
+def get_conda_test_env(test_env_name='jedi-test-env'):
```

Comment:
Please add a docstring to describe what this helper function does.

Reply:
Fixed